excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/validation/CategoryIdValidatorTest.java

```diff
@@ -0,0 +1,178 @@
+package uk.gov.hmcts.ccd.definition.store.excel.validation;
+
+import lombok.val;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception.InvalidImportException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.ParseContext;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseFieldEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CaseTypeEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.CategoryEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.FieldTypeEntity;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Redundant code `@ExtendWith(MockitoExtension.class)`

Reply:
Fixed in [6c59bbc](https://github.com/hmcts/ccd-definition-store-api/pull/1171/commits/6c59bbc4a4b15cfa2d8ed4a7d100b6b62fc34bc0)